_docker/drupal/drupal-filesystem/web/modules/custom/rhd_common/rhd_common.sitemap.inc

```diff
@@ -75,13 +84,18 @@ function build_subpage_link_entry($sub_page, $template)
 }
 
 /**
- * Returns an array af links which are for product nodes
- * @param array $links Links in the sitemap
- * @param array $product_ids node ids of products
- * @return array An array with the link entry for a product, the key is key in the $links array
+ * Returns an array af links which are for product nodes.
```

Comment:
... an array _of_ links ...

Reply:
Thanks for catching that.

I also agree that some of the coding standard changes are fairly worthless and pedantic. I am just trying to follow some coding standard, but it doesn't particularly matter to me that it be the Drupal coding standards. If you have other preferences, let me know. I don't want to step on your toes in any way. As long as we are improving readability and adding helpful comments/documentation as much as is reasonable, then I am a very happy camper.